### DISCUSSION TITLE
Can't a large molecular drug be simulated with metabolites?

### DISCUSSION THREAD

**Discussion by csungeun** [Open-Systems-Pharmacology/Forum#1572] [How-Tos]
hi, 

I want to simulate the process of metabolizing a large molecule drug and transforming it into a metabolite at the same time.

I tried it, but the concentration of the metabolite is zero.

If you set the molecule of the parent drug to be small, even a little metabolite will come out, but if it is a large molecule, the concentration is zero.

Or should this be built with Mobi?
[prodrug.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/12066064/prodrug.zip)

**Comment by Christoph27** [comment]
Hi, 

if you  define your drug to be a "large molecule", passive diffusion across cell membranes is set to zero.
You need to add a mechanism by which the drug enters the cell, e.g. target mediated uptake into the cell or an active transport.

Best,
Christoph

**Comment by TWendl** [comment]
or - unlike in your attached file - you could define a metabolization process in the plasma or interstitial space.
